### ISSUE TITLE
Redirect Request: Archiving Form 22-5495

### ISSUE THREAD

**Issue by jsimonVA** [department-of-veterans-affairs/va.gov-team#88975]
### Status
**[2024-09-13] [Fran]** Mikki has approved going forward with this. It will be refined on Monday 9/16 and included in the sprint started on 9/18.
**[2024-07-31] [Fran]** Jenny will let us know when we can proceed. I've tentatively tagged with Sprint 10.
**[2024-07-30] [Fran]** Moved to Next Refinement. I have a question out to Jenny asking if Aug 6th (the last day of our current sprint) is still the date they need this released.

### Instructions
- Requests must be submitted **AT LEAST** 2 weeks in advance.  Some requests take a significant amount of time and coordination to implement, so start the process as soon as you know you will need one. 
- This issue will be used from initial request through implementation to ensure all individuals working on this are notified of status updates.  Please do not create multiple issues to track differnt steps.
- It is your responsibility to notify VA stakeholders as appropriate.  

| POC | Github alias(es) |  Team
--- | --- | ---
| Product team point(s) of contact: | @alexchan-va @jsimonVA  | Cinda Quattrini, Alex Chan, Jarred Simon
| Content point of contact: | @aprocik1  | A Procik
| IA point of contact: | @jennymayoco  |

### Type of request
- [x] We are retiring or taking down a page and need to redirect the URL (complete redirect section)
- [ ] We are changing the URL of an existing page (complete redirect section)
- [ ] We need a custom vanity URL (complete vanity URL section)
- [ ] We are removing a temporary redirect  (add relevant background and link to original redirect request)

### Do you need Public Websites to implement this redirect?
- [x] Yes
- [ ] No - Please note: Redirect still needs approval from IA and PW will do PR review

### Implementation date
When does this request need to be live? **NOTE:** Redirects deploy M-Th. Friday launches should be avoided.
Choose one:
- [x] On a specific date, connected to a program or publishing requirement. **DATE: ~~6 August 2024~~** ASAP in Sprint 13. Lost when Jenny left.
- [ ] Any day prior to a deadline/latest possible date. **DEADLINE:** 
- [ ] Any day after an earliest possible date. **EARLIEST DATE:**
- [ ] During a specific date range: ***EARLIEST DATE TO LATEST DATE:***
- [ ] No hard timing requirements, publish when possible.

### Redirects 
Current URL  |  Redirect Destination or New URL
---  |  ---
~https://www.va.gov/education/apply-for-education-benefits/application/5495/introduction~ <br> https://www.va.gov/education/apply-for-education-benefits/application/5495/ and all child pages| ~https://www.va.gov/education/apply-for-education-benefits/application/1995/introduction~ <br> https://www.va.gov/education/apply-for-education-benefits/application/1995/
https://www.va.gov/find-forms/about-form-22-5495/ | https://www.va.gov/find-forms/about-form-22-1995/

7/25/24 - MN: Updated requested URLs - be sure to capture all child pages of the old form in the redirect, and redirect them to the canonical url of the 1995 so it is not later impacted by change in form flow.
7/25/24 - JM: Added about va form links

### Vanity URLs 
Vanity URL  |  Landing page
---  |  ---
requested URL | landing page URL

**Link to campaign landing page request issue:**
Jira Link -> https://jira.devops.va.gov/browse/VEBT-139

***Summary
The form 22-5495 is being merged into the form 22-1995. The new version (changes made) of the 22-1995 is set to go into prod on 6 August 2024. The Form 22-5495 will no longer exist, as Form 22-1995 is replacing it.

### Process, Roles and Responsibilities
- [ ] Requesting team: Above information is provided
- [ ] Requesting team: All appropriate VA stakeholders are notified as appropriate
- [x] IA: Request is vetted and documented and implementation plan is clear 
- [x] IA: Request is assigned to appropriate team for implementation 
- [ ] Implementation team: Work is complete
- [ ] Implementation team: Validated in production
- [ ] Requesting team: Validates everything is correct in production and closes ticket

**Comment by jennymayoco** [comment]
Hi @jsimonVA, thank you for submitting this ticket! A couple of follow-up questions:
1. Will you be working with Sitewide Content team to update the Drupal page, `https://www.va.gov/education/change-gi-bill-benefits/`?
2. Would we be able to add this form details page `https://www.va.gov/find-forms/about-form-22-5495/` to the redirect request?


cc @megzehn @strelichl for visibility. There might be some content work and would love to get your perspective on this

**Comment by jsimonVA** [comment]
Hi @jennymayoco, yes we will work to have the Drupal page updated but figured this could be handled after the redirect, but looks like we should start a request on that now. On the 1995 intro page, we are adding text that states that 1995 is now for both the 5495 and the 1995. 

Yes to question 2 as well. We will need to work with the Sitewide Content team to make sure we update the 1995 form details page to include the changes.

**Comment by jennymayoco** [comment]
Thanks for confirming! Would you be able to submit a CAIA intake ticket for that work? We would need to update any related content, links, and references currently on the site and then this redirect request would be the last step in this process.

cc @strelichl @NaomiPMC

**Comment by jennymayoco** [comment]
@jsimonVA There's been an update to the redirect process. For product work, the Public Websites team will not be handling these any longer and they won't be doing PR reviews. Product teams will be responsible for implementing the redirects themselves and testing. You would still need to submit a ticket, such as this one, for IA to review URLs, but Public Websites won't be handling the implementation part. We are currently working on documentation on this, but it's not published yet. In the meantime, please reach out to the Platform team if you have any questions on the implementation process.

cc @FranECross @jilladams @mnorthuis for visibility

**Comment by jsimonVA** [comment]
@jennymayoco i want to make sure I understand so I know what my next steps are. This ticket I have for redirecting the 5495 URL can not be accomplished by this team and I need to open a support ticket with Platform. Is that correct?

**Comment by mnorthuis** [comment]
@jsimonVA This ticket is required for review and approval of the redirect.  Your team will implement the redirect once reviewed and approved.  Platform can assist with any implementation questions you may have.

**Comment by jsimonVA** [comment]
@mnorthuis  Gotcha. Follow up question. does the team still have the same time line of 2 weeks for the review process? I only ask so that we can update our timeline as well when communicating to EDU/Stakeholders. Thanks.

**Comment by mnorthuis** [comment]
@jsimonVA no, it won't take that long to review.  This is a new process change for us so we still need to update the template and instructions.  

@jennymayoco can you add the redirect for the form finder detail page to this as well.  That will also need to be redirected.

**Comment by jsimonVA** [comment]
Just curious where we are with this request. Is it on me to start doing the redirects now or am I still waiting for the approval? Also, is there any documentation on how to complete the redirect ourselves?

**Comment by jennymayoco** [comment]
Hi @jsimonVA, thanks for checking in! A quick update on the redirect request. We're pausing on moving redirect implementation work from PW team to product teams until there is documentation and guidelines in place. So PW team will be assisting with this redirect request. I see you've also submitted a CAIA intake ticket, thank you for submitting that! I'm checking in with our team on any status updates on that work. An FYI that some of our Sitewide Content folks are out this week. As a reminder, we'll need to update content so folks know why they are being redirected to another page before implementing the redirect.

@jilladams @FranECross this redirect request is to archive a form. It's being replaced by form 22-1995 so we are redirecting the online application and the about va form details page. There's additional CAIA content tasks that need to be done before the redirect is implemented, but wanted to get this in your queue. Other than that, these are good to go once content has been updated. Let me know if you have any questions!

cc @NaomiPMC @strelichl

**Comment by jsimonVA** [comment]
Thank you @jennymayoco  for the update. I appreciate it.

**Comment by FranECross** [comment]
@jennymayoco Thanks for the info! I'll have this refined so that when content is good to go, we'll be able to schedule for the following sprint. cc @jilladams

**Comment by jennymayoco** [comment]
Hi @FranECross that date is when the new version of form 22-1995 goes to prod, if my understanding is correct @jsimonVA . We would still need the content updates before the redirect is implemented as this would be the last step in that process so unfortunately this most likely won't be ready by that date.

**Comment by FranECross** [comment]
@jennymayoco Thanks for the info, and no worries. I just wanted to be sure I understood timing, and if we were trying to coordinate closing the two actions. I'll tag for possibly being in Sprint 10, which is 08/07 to 08/20.

**Comment by mnorthuis** [comment]
@FranECross Is there a reason this redirect has not been implemented yet?

**Comment by FranECross** [comment]
@mnorthuis It's my understanding we're still awaiting word content is ready/approved. Jenny's last comment indicated it wasn't ready yet, and we've not had the go-ahead yet. https://github.com/department-of-veterans-affairs/va.gov-team/issues/88975#issuecomment-2261506030

**Comment by mnorthuis** [comment]
@FranECross Ok, it looks like this got lost when Jenny left.  This redirect can be completed as soon as you are able. Thank you!

**Comment by randimays** [comment]
@jsimonVA The redirects are in production and they seem to be working as expected. Please validate on your end and close the ticket if there aren't any issues. Thank you!

**Comment by jsimonVA** [comment]
Looks good to me. Thank you @randimays

**Comment by randimays** [comment]
Thanks! Closing per @jsimonVA's comment above.
